pages/news.vue

```diff
@@ -87,3 +121,13 @@ export default {
   }
 }
 </script>
+
+<style scoped>
+.active {
+  color: rgb(157, 157, 157);
+  margin: 8px 8px;
+}
+.inactive {
+  margin: 8px 8px;
+}
```

Comment:
Before customizing the styling, can we make use of the built-in "active" property of `v-chip` (https://vuetifyjs.com/en/api/v-chip/#props)

```html
<v-chip :active="selectedTags.include('hello')">Hello</v-chip>
```